common/src/main/java/com/leo/utils/LoadConfig.java

```diff
@@ -20,7 +20,7 @@ public static LoadConfig getInstance() throws FileNotFoundException, IOException
     if (instance == null) {
       synchronized (LoadConfig.class) {
         if (instance == null) {
-          instance = new LoadConfig("common/src/main/resources/config.properties");
+          instance = new LoadConfig("server/src/main/resources/config.properties");
```

Comment:
i think this is a mistake

Reply:
em để v thấy code chạy dc :vv fix bug tào lao ấy anh. anw cái loafconfig bên common cũng k quan trọng lắm